elementary/utils/time.py

```diff
@@ -6,9 +6,9 @@
 MILLISECONDS_IN_HOUR = (1000 * 60 * 60)
 
 
-def convert_utc_time_to_local_time(utc_time: 'datetime') -> 'datetime':
+def convert_utc_time_to_local_time(user_tz: str , utc_time: 'datetime') -> 'datetime':    
```

Comment:
I think that a better name will be `convert_utc_time_to_timezone`
and the params should be `utc_time` and an optional `timezone`:
```suggestion
def convert_utc_time_to_timezone(utc_time: 'datetime', timezone: Optional[str] = None) -> 'datetime':    
```